src/util/nativeStorageHelper.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable fp/no-class */
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+/**
+ *
+ */
+function handleError(func: string, param?: string): Promise<string> {
+  let message
+  if (!param) {
+    message = func
+  } else {
+    message = `${func}() requires at least ${param} as its first parameter.`
+  }
+  console.warn(message) // eslint-disable-line no-console
```

Comment:
`warn` is [set to allow]() so I am not sure why the linter is erroring here.

https://github.com/cybersemics/em/blob/22ca9a103482a38ebb8e0e6cfc205b35ce89c14d/package.json#L33